modules/rest-api/pages/rest-manage-log-collection.adoc

```diff
@@ -15,54 +15,235 @@ GET /pools/default/tasks
 
 * `tmpDir`.
+The pathname of the directory on each node into which temporary files are written, during log-collection.
+If a node does not have the pathname, the operating system's temporary directory is used instead.
+If the administrator-specified directory has insufficient space, or if the directory is not accessible by user `couchbase`, log-collection fails for that node.
 
 * `logRedactionSalt`.
+Used optionally, to specify a string to be used as the salt, when `partial` redaction is specified; so as to increase redaction-security.
+If no salt is specified, a salt is automatically generated and applied by Couchbase Server.
 
 * `upload-host`.
+Used optionally, to specify, as a URL, the `upload-host-name` to which the zip file containing the collected logs is to be uploaded.
 
 * `customer`.
+Used optionally, to specify a `customer-name` that is the name of the customer uploading logs.
 
 * `ticket`.
+Used optionally, to specify a `ticket-number` that corresponds to a ticket created by Couchbase Support to track the issue in connection with which logs are being uploaded.
+
+[#responses]
+== Responses
+
+On success, both `POST /controller/startLogsCollection` and `POST /controller/cancelLogsCollection` return `200 OK`; and an empty array.
+
+Status on an ongoing log-collection, including reports of failure, can be accessed by means of `GET /pools/default/tasks`.
+For example, starting log-collection with an inaccessible `output` directory returns `200 OK`: however, subsequent use of `GET /pools/default/tasks` returns output containing the following:
+
+[source,bourne]
+----
+"perNode": {
+      "ns_1@10.144.231.101": {
+        "status": "failed",
+        "collectionStatusCode": "1",
+        "collectionOutput": "do not have write access to the directory /home/vagrant/output2\n"
+      },
+          .
+          .
+          .
+----
+
+Failure to authenticate returns `401 Unauthorized`.
+Specifying an incorrect URL returns `404 Object Not Found`.
+An attempt to start log-collection while log-collection is already underway returns `400 Bad Request` and an error message such as `{"_":"Logs collection task is already started"}`.
+
+An attempt to start log-collection while specifying an unknown node returns `400 Bad Request` and an error message such as `{"nodes":"Unknown nodes: [\"ns_1@10.144.231.105\"]"}`.
+
+[#examples]
+== Examples
+
+The following example starts log-collection for three, explicitly specified cluster-nodes.
+It also specifies that partial redaction be used, and provides a salt.
+Additionally, it specifies an upload-host, a customer-name, and a ticket-number.
+
+[source,bourne]
+----
+curl -X POST http://10.144.231.101:8091/controller/startLogsCollection \
+-u Administrator:password \
+-d nodes=ns_1%4010.144.231.101%2Cns_1%4010.144.231.102%2Cns_1%4010.144.231.103 \
```

Comment:
Would you change %40 to @

Reply:
I have changed all the encoded elements to unencoded.